src/components/controllers/repl-page.jsx

```diff
@@ -24,11 +24,8 @@ export default function ReplPage() {
 					height: 100% !important;
 					overflow: hidden !important;
 				}
-				footer {
-					display: none !important;
-				}
```

Comment:
No footer on the REPL of course -- I had just copy/pasted this CSS, it's not relevant anymore.